scripts/gff_metaparser/gffstruct/rules/alias.py

```diff
@@ -14,63 +14,67 @@
 # limitations under the License.
 
 
-# 
+#
```

Comment:
that was a meaningful space! the whole comment made obsolete :(
(don't fix with this PR, please :)

Reply:
What is that supposed to mean?

Reply:
no idea

Reply:
mb. "TODO: add some doc"